src/ott/geometry/costs.py

```diff
@@ -36,11 +36,10 @@ class CostFn(abc.ABC):
   that function is split into two norms -- evaluated on each input separately --
   followed by a pairwise cost that involves both inputs, as in:
 
-  .. math::
-
-    c(x,y) = norm(x) + norm(y) + pairwise(x,y)
+  `c(x,y) = norm(x) + norm(y) + pairwise(x,y)`
```

Comment:
Not a huge issue, but can we render this somehow nicely?

Reply:
I want to avoid the classic Latex bug of having the parser interpret `norm` as if it was n times o times r times m etc... WDYT not?